### PR TITLE
tests and functionality

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@miltontony @justinvdm ready for review

**Comment by codiebeulaine** [comment]
@saeedmb8 ready for review also

File: tuneme/templates/core/article_page.html [hunk 1]
```diff
@@ -63,6 +63,17 @@
 </div>
 </div>
 
+{% has_questions self as questions %}
+{% if questions %}
+<div class="block {{self.get_effective_extra_style_hints|default:"orange"}}">
```
Comment (smarzban) [inline tuneme/templates/core/article_page.html]: In an article level you want to get the colour from it's parent so you should do `self.get_parent_section.get_effective_extra_style_hints`

File: tuneme/templates/polls/poll_page.html [hunk 2]
```diff
@@ -2,12 +2,16 @@
...
 	{% if questions %}
 		{% for question in questions %}
+		{% if forloop.first%}
 			<h2>{% trans "What's your opinion?" %}</h2>
-			<div class="block orange">
+			{% endif %}
+			<div class="block {{question.get_effective_extra_style_hints|default:"orange"}}">
+				{% if forloop.first %}
 				<div class="title">
 					<div class="icon"><a href="{% url 'molo.polls:results' question.id %}"><img src="{% static 'img/polls.png' %}" alt="Have Your Say"></a></div>
```
Comment (smarzban) [inline tuneme/templates/polls/poll_page.html]: Please remove the link here as well.

**Comment by smarzban** [comment]
:+1:

**Pull request merged**